sktime/transformations/bootstrap/__init__.py

```diff
@@ -0,0 +1,9 @@
+# -*- coding: utf-8 -*-
+"""Bootstrapping methods for time series."""
```

Comment:
perhaps explain here, and in the docs, that this takes a single series and produces a panel of series?